bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/Utils.java

```diff
@@ -74,9 +75,13 @@ public static ZonedDateTime parseDate(String date) throws DateTimeParseException
      * @return
      */
     public static String formatDate(Instant date) {
-        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
-                .withZone(ZoneId.systemDefault());
-        return formatter.format(date);
+        try {
```

Comment:
If seems this method is only used for logging. If you would explicitly handle it here, there would be no need for the `Instant.EPOCH` work-around:
```suggestion
        if (date == OUTDATED_DATE) {
            return "(expired)";
        }
        try {
```

Reply:
it is only used for logging, yes. But still the dates which are used for input also come from the API and thus this method should stop the binding if there is unexpected input.